### PR TITLE
added reload prefix to tls cmd flags to avoid collision

### LINKED ISSUE #1072: config-reloader v0.47.0 crashes
After v0.47.0 config-reloader crashes with following error:

```
/app flag redefined: tlsCertFile
panic: /app flag redefined: tlsCertFile

goroutine 1 [running]:
flag.(*FlagSet).Var(0x400017a000, {0x14f1a78, 0x4000386140}, {0x123cc6a, 0xb}, {0x12a564a, 0x57})
	/usr/local/go/src/flag/flag.go:1028 +0x2cc
flag.(*FlagSet).StringVar(...)
	/usr/local/go/src/flag/flag.go:879
flag.(*FlagSet).String(0x400017a000, {0x123cc6a, 0xb}, {0x0, 0x0}, {0x12a564a, 0x57})
	/usr/local/go/src/flag/flag.go:892 +0x98
flag.String(...)
	/usr/local/go/src/flag/flag.go:899
main.init()
	/workspace/cmd/config-reloader/main.go:61 +0x538
```

 Bug was introduced at f3efa3f1bdd2b57d4fc6962e15e3c0f7ed3faff7

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
fixes #1072

### CODE REVIEW

File: cmd/config-reloader/main.go [hunk 1]
```diff
@@ -56,15 +56,15 @@ var (
 	webhookMethod = flag.String(
 		"webhook-method", "GET", "the HTTP method url to use to send the webhook")
 
-	tlsCaFile = flag.String("tlsCAFile", "",
+	tlsCaFile = flag.String("reload.tlsCAFile", "",
```
Comment (f41gh7) [inline cmd/config-reloader/main.go:59]: Please mention changes at change log.

Comment (AndrewChubatiuk) [reply cmd/config-reloader/main.go:59]: done

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks for contribution!

**Pull request merged**